content/intro-to-storybook/react-native/en/simple-component.md

```diff
@@ -75,167 +125,108 @@ There are two basic levels of organization in Storybook: the component and its c
+          />
         ) : (
-          <PercolateIcons name="check" size={20} color={'#2cc5d2'} />
+          <MaterialIcons name="check-box" size={24} color="#26c6da" />
         )}
       </TouchableOpacity>
       <TextInput
         placeholder="Input Title"
-        style={
-          state === 'TASK_ARCHIVED' ? styles.ListItemInputTaskArchived : styles.ListItemInputTask
-        }
         value={title}
         editable={false}
+        style={
+          state === "TASK_ARCHIVED"
+            ? styles.listItemInputTaskArchived
+            : styles.listItemInputTask
+        }
       />
       <TouchableOpacity onPress={() => onPinTask(id)}>
-        <PercolateIcons
+        <MaterialIcons
           name="star"
-          size={20}
-          color={state !== 'TASK_PINNED' ? '#eee' : '#26c6da'}
+          size={24}
+          color={state !== "TASK_PINNED" ? "#eee" : "#26c6da"}
         />
       </TouchableOpacity>
-    </SafeAreaView>
+    </View>
   );
-}
-```
-
-The additional markup from above combined with the styling we created earlier yields the following UI:
-
-<video autoPlay muted playsInline loop>
-  <source
-    src="/intro-to-storybook/finished-task-states.mp4"
-    type="video/mp4"
-  />
-</video>
-
-## Specify data requirements
-
-It’s best practice to use `propTypes` in React to specify the shape of data that a component expects. Not only is it self documenting, it also helps catch problems early.
-
-```js:title=components/Task.js
-import * as React from 'react';
-import PropTypes from 'prop-types';
-
-export default function Task({ task: { id, title, state }, onArchiveTask, onPinTask }) {
-  // ...
-}
-
-Task.propTypes = {
-  task: PropTypes.shape({
-    id: PropTypes.string.isRequired,
-    title: PropTypes.string.isRequired,
-    state: PropTypes.string.isRequired,
-  }),
-  onArchiveTask: PropTypes.func,
-  onPinTask: PropTypes.func,
 };
 ```
 
-Now a warning in development will appear if the Task component is misused.
+When you're done, it should look like this:
 
-<div class="aside">
-An alternative way to achieve the same purpose is to use a JavaScript type system like TypeScript to create a type for the component properties.
-</div>
+![a gif showing the task component in storybook](/intro-to-storybook/react-native-task-component-completed.gif)
 
 ## Component built!
 
-We’ve now successfully built out a component without needing a server or running the entire frontend application. The next step is to build out the remaining Taskbox components one by one in a similar fashion.
+We’ve now successfully built out a component without needing a server or running the entire application. The next step is to build out the remaining Taskbox components one by one in a similar fashion.
 
 As you can see, getting started building components in isolation is easy and fast. We can expect to produce a higher-quality UI with fewer bugs and more polish because it’s possible to dig in and test every possible state.
```

Comment:
End the chapter with an aside to tell the readers to commit their changes to their GitHub repo.